src/providers/azure-ad.js

```diff
@@ -0,0 +1,24 @@
+export default (options) => {
+  const tenant = options.tenantId ? options.tenantId : 'common'
```

Comment:
How about:
```suggestion
  const tenant = options.tenantId ?? 'common'
```

Reply:
I had no idea that 'suggested change' functionality existed 🤯

Reply:
you can even batch my suggestions into a single commit and apply them 😉

https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes